bad_storage.go

```diff
@@ -17,6 +17,7 @@ var _ storage.ClientImpl = badStorage{}
 func (bs badStorage) OpenTorrent(*metainfo.Info, metainfo.Hash) (storage.TorrentImpl, error) {
 	return storage.TorrentImpl{
 		Piece: bs.Piece,
+		Flush: func() error { return nil },
```

Comment:
I think it's common enough not to have a Flush method, that teh caller can check for Flush != nil in the TorrentImpl struct.

Reply:
done